File: javascript/chapter7/movie_tickets.js
while (true) {
	const age = prompt("Enter the age: ");

	if (Number.isNaN(age)) {
		alert("Enter a valid age!!");
		continue;
	}


	if (Number(age) < 3) {
		alert("Movie ticket is free!");
	} else if (Number(age) >= 3 && Number(age) <= 12) {
		alert("Movie ticket is $10.");
	} else if (Number(age) > 12) {
		alert("Movie ticket is $15.");
	}
}

File: python/chapter7/movie_tickets.py
while True:
    try:
        age = int(input("Enter the age: "))
    except ValueError:
        print("Enter a valid age!!\n")
    else:
        if age < 3:
            print("Movie ticket is free!")
        if age >= 3 and age <= 12:
            print("Movie ticket is $10.")
        if age > 12:
            print("Movie ticket is $15.")
